borg/helpers.py

```diff
@@ -18,9 +18,23 @@
 
 import msgpack
 
-from . import hashindex
-from . import chunker
-from . import crypto
+def detect_cython():
+    """allow for a way to disable Cython includes
+
+    this is used during usage docs build, in setup.py. It is to avoid
+    loading the Cython libraries which are built, but sometimes not in
+    the search path (namely, during Tox runs).
+
+    we simply check an environment variable (``BORG_CYTHON_DISABLE``)
+    which, when set (to anything) will disable includes of Cython
+    libraries in key places to enable usage docs to be built.
+    """
+    return os.environ.get('BORG_CYTHON_DISABLE')
```

Comment:
seems like a logic inversion here.
name: detect_cython
returns True if we don't have cython / don't want to load cython stuff?